--- configure	2009-05-10 12:08:26.000000000 +0200
+++ configure.new	2013-06-19 20:07:34.372619000 +0200
@@ -20478,6 +20478,26 @@
 
 # Checks for libraries.
 
+# Check whether --with-mpfr was given.
+if test "${with_mpfr+set}" = set; then
+  withval=$with_mpfr;
+      CPPFLAGS="$CPPFLAGS -I$withval/include"
+       LDFLAGS="$LDFLAGS -L$withval/lib"
+       mpfr_lib_path="$withval/lib"
+
+fi
+
+
+
+# Check whether --with-gmp was given.
+if test "${with_gmp+set}" = set; then
+  withval=$with_gmp;
+      CPPFLAGS="$CPPFLAGS -I$withval/include"
+       LDFLAGS="$LDFLAGS -L$withval/lib"
+       gmp_lib_path="$withval/lib"
+
+fi
+
 { echo "$as_me:$LINENO: checking for __gmpz_init in -lgmp" >&5
 echo $ECHO_N "checking for __gmpz_init in -lgmp... $ECHO_C" >&6; }
 if test "${ac_cv_lib_gmp___gmpz_init+set}" = set; then
@@ -20630,25 +20650,6 @@
 
 
 
-# Check whether --with-mpfr was given.
-if test "${with_mpfr+set}" = set; then
-  withval=$with_mpfr;
-      CPPFLAGS="$CPPFLAGS -I$withval/include"
-       LDFLAGS="$LDFLAGS -L$withval/lib"
-       mpfr_lib_path="$withval/lib"
-
-fi
-
-
-
-# Check whether --with-gmp was given.
-if test "${with_gmp+set}" = set; then
-  withval=$with_gmp;
-      CPPFLAGS="$CPPFLAGS -I$withval/include"
-       LDFLAGS="$LDFLAGS -L$withval/lib"
-       gmp_lib_path="$withval/lib"
-
-fi
 
 
 
