--- configure.ac	2013-01-25 13:53:43.000000000 +0100
+++ configure.ac2	2013-05-24 16:44:05.759199000 +0200
@@ -25,12 +25,6 @@
 AC_PROG_MAKE_SET
 
 # Checks for libraries.
-AC_CHECK_LIB(gmp, __gmpz_init, , [AC_MSG_ERROR(
-[GNU MP not found, see http://gmplib.org])])
-AC_CHECK_LIB(mpfr, mpfr_add, , [AC_MSG_ERROR(
-[MPFR not found, see http://www.mpfr.org])])
-
-
 AC_ARG_WITH(mpfr,
    [  --with-mpfr=DIR          MPFR install directory ], [
       CPPFLAGS="$CPPFLAGS -I$withval/include"
@@ -45,6 +39,11 @@
        gmp_lib_path="$withval/lib"
    ])
 
+AC_CHECK_LIB(gmp, __gmpz_init, , [AC_MSG_ERROR(
+[GNU MP not found, see http://gmplib.org])])
+AC_CHECK_LIB(mpfr, mpfr_add, , [AC_MSG_ERROR(
+[MPFR not found, see http://www.mpfr.org])])
+
 
 # Checks for header files.
 AC_HEADER_STDC
