--- Makefile.am	2013-05-30 09:50:42.000000000 +0200
+++ ../../libfplll-4.0.1/src/Makefile.am	2012-01-19 12:32:01.000000000 +0100
@@ -39,7 +39,7 @@
 EXTRA_libfplll_la_SOURCES=gso.cpp gso.h lll.cpp lll.h wrapper.cpp wrapper.h \
 	bkz.cpp bkz.h enumerate.cpp enumerate.h svpcvp.cpp
 libfplll_la_LIBADD=-lgmp -lmpfr
-libfplll_la_LDFLAGS=-no-undefined -version-info 1:0:1
+libfplll_la_LDFLAGS=-version-info 1:0:1
 
 libfplllv_la_SOURCES=$(libfplll_la_SOURCES)
 EXTRA_libfplllv_la_SOURCES=$(EXTRA_libfplll_la_SOURCES)
